NicolaeChristof/Descend-Into-Darkness
# Feature wall climb

Hey guys I think this really adds a lot, I hope you guys enjoy it. This PR is for the ledge grab. The way it works is any time the player encounters a grabable ledge (a collider is searching for ledges with a specific trace channel) the player will automagically grab onto it, from the hanging position the player can either press W to climb up onto the ledge or S to fall down.

How to test:
- jump onto any ledge
- Make sure you can pull yourself up onto the ledge with W while hanging
- Make sure you can drop down with S while hanging

No known bugs
